src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,26 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
```

Comment:
Hmm, well either way, now we have coverage over the case; and it's not an issue.